docs/go/how-to-set-tls-configuration-in-go.md

```diff
@@ -0,0 +1,26 @@
+---
+id: how-to-set-tls-configuration-in-go
+title: How to set TLS configuration in Go
+sidebar_label: Set TLS configuration
+description: Set TLS configuration
```

Comment:
I'd clarify that this is for mTLS, for server side TLS only there are different options.